media-sound/helm/helm-9999.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils
+
+DESCRIPTION="A free polyphonic synth with lots of modulation"
+HOMEPAGE="http://tytel.org/helm/"
+LICENSE="GPL-3+ CC-BY-3.0"
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/mtytel/helm.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/mtytel/helm/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+SLOT="0"
+
+IUSE="standalone lv2 vst"
+REQUIRED_USE="|| ( standalone lv2 vst )"
```

Comment:
Yeah that is Ok, i guess. But why is VST not a "sane default". I like to test as much as possible, for a small price of course. If you have to make a lot of special adjustments to test a small edge-case, that's not a good relation between workload and benefit. But here it looks like you just have to add 3 letters.